Post.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'; 
+import CommentsList from './CommentsList.js';
+
+function Post(props) {
+  const {
+    title: postTitle,
+    body: postBody,
+    userName: postAuthorName,
+    userEmail: postAuthorEmail,
+    userAddress: postAuthorAddress,
```

Comment:
you do not really need to rename all properties to names with "post", because of you already in Post component

it will be shorter and more readable